test_requirements.txt

```diff
@@ -6,3 +6,4 @@ freezegun==0.3.12
 Faker==1.0.1
 django-reversion==4.0.2
 skip-django-germanium==2.4.0
+django-auditlog
```

Comment:
What's the reasoning behind the decision to not pin the version?

Reply:
There isn't any concern for incompatibility in this library, if there will be, we can at least add constraint to setup.py.